modules/gitops-release-notes-1-18-0.adoc

```diff
@@ -29,20 +29,121 @@ $ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-
+** A new `Enabled` checkbox in the `Application Create` and `Application Summary` panels.
+** Auto-sync can be enabled or disabled independently of `prune` and `selfHeal`.
+** Defaults to `true` for backward compatibility.
+** When set to `false`, auto-sync is explicitly disabled even if the automated block exists.
++
+[source,YAML]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: my-app
+  namespace: argocd
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/my-org/my-app.git
+    targetRevision: HEAD
+    path: manifests
+  destination:
+    server: https://kubernetes.default.svc
+    namespace: my-app-namespace
+  syncPolicy:
+    automated:
+      enabled: false # Explicitly disables auto-sync
+      prune: true
+      selfHeal: true
+----
++
+[NOTE]
+====
+If the `enabled` field is not set, automated synchronization defaults to `enabled` to maintain backward compatibility.
+====
++
+link:https://issues.redhat.com/browse/GITOPS-5430[GITOPS-5430]
+
+Granular control of banner display in the Argo CD UI::
+With this update, the {gitops-title} Operator adds support for the permanent and position properties within the `spec.banner` field in the Argo CD CR. The following options provide more granular control over how banners are displayed in the Argo CD web UI.
+* `spec.banner.permanent`: A boolean field, when set to `true`, prevents users from dismissing the banner.
+* `spec.banner.position`: A string field that specifies where the banner is displayed. Supported values are `top` and `bottom`.
++
+[source,YAML]
+----
+spec:
+  banner:
+    content: "Maintenance in progress"
+    permanent: true
+    position: top
+----
++
+link:https://issues.redhat.com/browse/GITOPS-6679[GITOPS-6679]
+
+Upgraded Kubernetes and Argo CD dependencies::
+With this update, the {gitops-title} Operator upgrades the Kubernetes and Argo CD dependencies used in the {gitops-shortname} backend. The newer versions address vulnerability alerts previously flagged by CVE scanners in older dependency versions, ensuring alignment with more recent and secure libraries.
++
+link:https://issues.redhat.com/browse/GITOPS-6860[GITOPS-6860]
+
+Default resource exclusions in {gitops-title}::
+With this update, the {gitops-title} Operator introduces a default `resource.exclusions` configuration, consistent with upstream Argo CD. By default, the Argo CD Application Controller excludes well-known resources created by controllers and frequently changing or auto-generated resources, which reduces unnecessary load. You can override or customize these exclusions through the Argo CD configuration if needed.
++
+link:https://issues.redhat.com/browse/GITOPS-7434[GITOPS-7434]
+
 [id="fixed-issues-1-18-0_{context}"]
 == Fixed issues
 
+Fixed misleading error logs when the `ConsoleLink` resource is disabled::
```

Comment:
I am wondering if this needs to be mentioned or not, I was unable to reproduce this behaviour

cc: @nmirasch

Reply:
I’m not completely sure either. My conclusion was that I wasn’t able to modify that env var to disable the console link. From the code, if the env var is set to true, the link is removed and this error should appear even it continues working fine. That said, I didn’t see this log in previous versions, but I don’t think we can conclude that nobody sees it. For me it’s fine to remove this from the release notes.